x-pack/plugins/translations/kibana.json

```diff
@@ -1,5 +1,9 @@
 {
   "id": "translations",
+  "owner": {
+      "name": "Kibana Localization",
+      "githubTeam": "kibana-localization"
```

Comment:
We don't have `x-pack/plugins/translations` assigned to the localization team in the `CODEOWNERS` file -- but I assume we are explicitly excluding it so we aren't pinged every single time a team adds or updates a string. Either way makes sense to add it here 👍